h2o-core/src/test/java/water/rapids/RapidsTest.java

```diff
@@ -688,13 +688,20 @@ static boolean checkSaneFrame_impl() {
   }
 
   @Test public void test_frameKeyStartsWithNumber() {
-    Frame fr = parse_test_file(Key.make("123STARTSWITHDIGITS"), "smalldata/logreg/prostate.csv");
+    Frame fr = null;
+    Key key = null;
+    
     try {
+      Scope.enter();
```

Comment:
Rapids don't use `Scope` AFAIK. Because you don't explicitly track any keys using Scope in this test, adding Scope.enter and exit is redundant.